lib/orogen/gen/project.rb

```diff
@@ -1114,6 +1114,70 @@ def external_task_context(name, **options, &block)
                     new_task
                 end
 
+                # Sets the c++ standard to be used when parsing headers using
+                # import_types_from. The standard should be a compiler flag
+                # without the -std= prefix. nil will clear the set standard.
+                def set_cxx_standard(standard)
```

Comment:
We don't use `set_` or `get_` prefixes in Ruby. For the DSL, Just `cxx_standard` or `minimum_cxx_standard`, that would be both the getter (if without arguments) or the setter (with arguments).

There is a `dsl_attribute` helper for this, 

```
dsl_attribute :cxx_standard do |new_value|
   # Process the new value
end
```